character.cpp

```diff
@@ -2,14 +2,14 @@
 
 
 
-Character::Character(const std::string& name, int health, const int& damage) :name(name), health(health), damage(damage) {}
-std::string Character::getName() const { return name; }
+Character::Character(std::string name, int health, int damage) :name(name), health(health), damage(damage) {}

```

Comment:
A `name` jó volt const refként, csak az intnél felesleges. Az került be itt hibásan: 5ae4e13 A string továbbra is legyen const referenciával átadva, mert összetett típus!